pennylane_lightning/lightning_qubit/lightning_qubit.py

```diff
@@ -197,7 +197,7 @@ def __init__(
             kernel_name="Local",
             num_burnin=100,
             batch_obs=False,
-        ):
+        ):  # pylint: disable=too-many-arguments
```

Comment:
Same as ``lightning_kokkos``. Do we need to add this?

Reply:
Removed.